pkgs/applications/science/electronics/qcsxcad/default.nix

```
@@ -0,0 +1,52 @@
+{ stdenv
+, mkDerivation
+, fetchFromGitHub
+, cmake
+, csxcad
+, tinyxml
+, vtkWithQt5
+, qt5
```

you should import the individual packages from the qt5 package set
```suggestion
, wrapQtAppsHook
```